Packs/CybleEventsV2/Integrations/CybleEventsV2/CybleEventsV2.py

```diff
@@ -94,15 +288,213 @@ def test_response(client, method, base_url, token):
         base_url: base url for the server
         token: API access token
 
-    Returns: test response
+    Returns: test response
+
+    """
+    fetch = fetch_subscribed_services(client, method, base_url, token)
+    if fetch:
+        return 'ok'
+    else:
+        demisto.error("Failed to connect")
+        raise Exception("failed to connect")
+
+
+def cyble_events(client, method, token, url, args, base_url, last_run, hide_cvv_expiry, skip=True):
+    """
+    Fetch alert details from server for creating incidents in XSOAR
+    Args:
+        last_run: get last run details
+        base_url: base url for subscribed services
+        client: instance of client to communicate with server
+        method: Requests method to be used
+        token: API access token
+        url: end point URL
+        args: input args
+        hide_cvv_expiry: hide expiry / cvv number from cards
+        skip: skip the validation for fetch incidnet
+
+    Returns: events from the server
+
+    """
+
+    input_params = {}
+
+    input_params['order_by'] = args.get('order_by', "asc")
```

Comment:
in the yml file the default value is 'desc', it should be the same.

Reply:
Fixed it